charts/ccsm-helm/README.md

```diff
@@ -2,7 +2,24 @@
 
 # Camunda Cloud Self-Managed Helm Chart
 
-This functionality is in beta and is subject to change. The design and code is less mature than official GA features and is provided as-is with no warranties. Beta features are not subject to the support SLA of official GA features.
+> This functionality is in beta and is subject to change. The design and code is less mature than official GA features and is provided as-is with no warranties. Beta features are not subject to the support SLA of official GA features.
+
+- [Camunda Cloud Self-Managed Helm Chart](#camunda-cloud-self-managed-helm-chart)
+  * [Requirements](#requirements)
+  * [Installing](#installing)
+  * [Configuration](#configuration)
+    + [Global](#global)
+    + [Zeebe](#zeebe)
+    + [Zeebe-gateway](#zeebe-gateway)
```

Comment:
```suggestion
    + [Zeebe Gateway](#zeebe-gateway)
```

Reply:
I used the name from the configs but yeah might be not so optimal